src/lib/drivers/device/Device.hpp

```diff
@@ -165,7 +166,7 @@ class __EXPORT Device
 	 */
 	struct DeviceStructure {
 		DeviceBusType bus_type : 3;
-		uint8_t bus: 5;    // which instance of the bus type
+		uint8_t bus: 6;    // which instance of the bus type
```

Comment:
Why does this need to increase?

Reply:
Doesn't bus_type need to increase to 4 bits and then bus shrink to fit?

Reply:
@dagar I updated the PR and removed this change. As far as I understand DeviceBusType is 3 bits which should be enough since the number of elements is only 6 (prev 5).